### ISSUE TITLE
iOS 17 update / Bluetooth

### ISSUE THREAD

**Issue by JCoulter85** [Flipper-XFW/Xtreme-Firmware#401]
### Describe the bug.

When you open the YouTube & TikTok application, the pervious and next buttons do not work.

### Reproduction

Power on. 
center circle button
Applications
Bluetooth>Bluetooth Remote>TiTok/YT Shorts

**Comment by JCoulter85** [comment]
Any update on this?

**Comment by Flipdaworld** [comment]
I have the same issue and I really like that feature seem like nobody has the solution

**Comment by Brereid98** [comment]
seems like it's due to an iOS change, the app uses mouse inputs to control TikTok which the phone doesn't seem to like, if you turn on Assistive Touch in the accessibility settings it allows mouse control on the iPhone and gets it working again

**Comment by Flipdaworld** [comment]
@Brereid98 man.... thank you thank you thank you , Yu area legend brother  it's back working again

**Comment by Brereid98** [comment]
You're welcome, happy to help

**Comment by Willy-JL** [comment]
can you try on latest release v52?

**Comment by Brereid98** [comment]
still doesn't seem to work unless assistive touch is on

**Comment by Flipdaworld** [comment]
Yeah sucks Sent from my iPhoneOn Dec 10, 2023, at 3:07 PM, Brereid98 ***@***.***> wrote:﻿
still doesn't seem to work unless assistive touch is on

**Comment by ghost** [comment]
it won't work without assistive touch, so just enable it. iOS doesn't have BT mouse support without Assistive Touch on.

**Comment by Flipdaworld** [comment]
It did work without it at one point , when I first got my flipper I never used assistive touch to get that to work but now I have to use it for it to work
